apps/api/controllers/players/destroy.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module Controllers
+    module Players
+      class Destroy
+        include Api::Action
+
+        def initialize(interactor: Interactors::Players::DestroyPlayer.new)
+          @interactor = interactor
+        end
+
+        def call(_params)
+          uuid = request.env['HTTP_UUID']
+
+          player = @interactor.call(uuid).player
+          halt 400 && self.status = 400 if player.nil?
+        end
+      end
+    end
+  end
+end
```

Comment:
How will this be used? Are players deleted after a game?

Reply:
@jgayfer Yeah, that's the goal; if we had websockets we could track all the connections and ping from React when a game is finished etc. but without them we need a way of deleting from the "lobby". With an API we can instead have the client remove itself when it's done playing, or by default have the server remove a player after 30 seconds of inactivity for example.

Reply:
resolved over slack